stringset.go

```diff
@@ -37,17 +37,17 @@ func (s *StringSet) All() []string {
 	return l
 }
 func (s *StringSet) Clear() {
-	s.strMap = make(map[string]bool)
+	s.strMap = make(map[string]struct{})
 }
-func (s *StringSet) Raw() map[string]bool {
+func (s *StringSet) Raw() map[string]struct{} {
```

Comment:
This breaks the interface for a few of my other projects, but it should be easy to fix!